src/etl/ETLService.cpp

```diff
@@ -110,11 +111,7 @@ ETLService::monitor()
         }
         catch (std::runtime_error const& e)
```

Comment:
Catching `runtime_error` here also would catch the ones we throw ourselves. This may or may not be desired. 
Certainly the `e.what()` part should still be logged - will fix.